src/D2L.Security.OAuth2/Keys/Default/MonoRsaSecurityKey.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace D2L.Security.OAuth2.Keys.Default {
+
+	internal sealed class MonoRsaSecurityKey : AsymmetricSecurityKey {

```

Comment:
ah yeah, looks like I did that everywhere but 1 place. Not sure why I thought it was different. Alright this is done.